server/lib/routes/get-verify-email.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const got = require('got');
+
+const logger = require('mozlog')('server.get-verify-email');
+const config = require('../configuration');
+
+module.exports = function (uid, code, options) {
+
+  return {
+    method: 'get',
+    path: '/verify_email/',
+    process: function (req, res, next) {
+      req.url = '/';
+      console.log('items', req.query);
+
+      let data = {
+        code: req.query.code,
+        uid: req.query.uid
+      };
+
+      let options = req.query.options;
```

Comment:
@divyabiyani these won't be `options` but `req.query.service` and `req.query.reminder`